funlet-find-me/tests/funlet-find-me.test.js

```diff
@@ -72,133 +70,68 @@ const EXAMPLE_CONTEXT = {
   PATH: '/funlet-find-me',
 };
 
-const DEFAULT_WHISPER_URL = EXAMPLE_BASE_URL + '?Whisper=true';
-const XML_DEFAULT_WHISPER_URL = EXAMPLE_BASE_URL + '?Whisper=true';
-const XML_WHISPER_URL_WITH_CUSTOM_MESSAGE =
-  XML_DEFAULT_WHISPER_URL + '&amp;Message=' + CUSTOM_MESSAGE_ENCODED;
+const DEFAULT_WHISPER_URL = `${EXAMPLE_BASE_URL}?Whisper=true`;
+const XML_DEFAULT_WHISPER_URL = `${EXAMPLE_BASE_URL}?Whisper=true`;
+const XML_WHISPER_URL_WITH_CUSTOM_MESSAGE = `${XML_DEFAULT_WHISPER_URL}&amp;Message=${CUSTOM_MESSAGE_ENCODED}`;
 
 const XML_DECLARATION = '<?xml version="1.0" encoding="UTF-8"?>';
 
 const FULL_RESPONSE_FIND_ME_1_1 =
-  XML_DECLARATION +
-  '<Response>' +
-  '<Dial ' +
-  'action="' +
-  EXAMPLE_BASE_URL +
-  '?Dial=true' +
-  '&amp;PhoneNumbers%5B0%5D=' +
-  PHONE_NUMBER2 +
-  '&amp;PhoneNumbers%5B1%5D=' +
-  PHONE_NUMBER3 +
-  '" ' +
-  'timeout="' +
-  DEFAULT_TIMEOUT +
-  '"' +
-  '>' +
-  '<Number url="' +
-  XML_DEFAULT_WHISPER_URL +
-  '">' +
-  PHONE_NUMBER1 +
-  '</Number>' +
-  '</Dial>' +
-  '</Response>';
+  `${XML_DECLARATION}<Response>` +
```

Comment:
all these multiline strings can be written to use a single starting \` and an ending `